src/components/util/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -108,10 +107,6 @@ const AutocompleteInput = React.forwardRef((props: Props, ref: any) => {
 			e.preventDefault();
 		}
 
-		if (value.trim().length > 0 && submitKeyCodes.includes(e.keyCode)) {
```

Comment:
you should return this, because submitKeyCodes contained KeyCodes.Tab , and right now we cant write name of needed session and press on TAB to submit it, as it was before